<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(/Diaimage/MainUI/Main3.bmp);</string>
    </property>
    <widget class="QTableWidget" name="mainList">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>311</width>
       <height>210</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/list.bmp);</string>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Device Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Addr</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="mainLight">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>112</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/Light_button.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mainTv">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>112</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/TV_button.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mainReg">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>218</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/Reg_button.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mainAir">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>218</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/Air_button.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mainStart">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>330</y>
       <width>210</width>
       <height>78</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Select.bmp);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="zTvWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/Main3_tv.bmp);</string>
     </property>
     <widget class="QPushButton" name="Back">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Back.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Power">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Power.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="tvChUp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_ch_up.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="tvChDown">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_ch_dw.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="tvVoDown">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>260</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_vo_dw.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="tvVoUp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_vo_up.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="tvImage">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>325</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/MainUI/Tv.bmp);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="zLightWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/Main3_light.bmp);</string>
     </property>
     <widget class="QPushButton" name="Back_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Back.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Power_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Power.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="lightBrUp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Bright_up.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="lightBrDown">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Bright_dw.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="lightImage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>140</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/MainUI/Light.bmp);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="zRefWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/Main3_tv.bmp);</string>
     </property>
     <widget class="QPushButton" name="Back_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Back.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Power_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Power.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="refTemUp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_temp_up.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="refTemDown">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_temp_dw.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="refImage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>144</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/MainUI/Ref.bmp);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="zAirWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(/Diaimage/MainUI/Main3_tv.bmp);</string>
     </property>
     <widget class="QPushButton" name="Back_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Back.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Power_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>330</y>
        <width>210</width>
        <height>78</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_Power.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="airTempUp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_temp_up.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="airTempDown">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_temp_dw.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="airPowerUp">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_wind_up.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="airImage">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>84</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/MainUI/Air.bmp);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="airPowerDown">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>260</y>
        <width>200</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(/Diaimage/DeviceUI/Device_Main_wind_dw.bmp);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
